report-to-row.ts

```diff
@@ -0,0 +1,60 @@
+import csvParse from 'csv-parse/lib/sync';
+
+export const reportToRowHeaders = (csvFileContents: string) => {
+  const singleReportRows: LighthouseCSVReportRow[] | undefined = csvParse(
+    csvFileContents,
+    {
+      columns: true,
+      skip_empty_lines: true,
+      ltrim: true,
+      relax: true, // https://csv.js.org/parse/options/
+    }
+  );
+  if (!singleReportRows || singleReportRows.length === 0) return false;
```

Comment:
Unfortunately this didn't show in the diff, but I added this check, since the other time the CSV is being parsed a few lines down, it says that the data might not exist, so I thought we should check that here too.